### DISCUSSION TITLE
Make menus keyboard accessible

### DISCUSSION THREAD

**Discussion by Fusseldieb** [directus/directus#2978] [Feature Requests]
I've noticed that in Directus v9.0.0-rc12 when I'm currently with a dropdown open, I can't simply press "G" on the keyboard and it jumps to "German" (for example).

In v8.1.1 Directus still used a native browser dropdown, which obviously supported "key shortcuts", but since v9 it has been replaced by a soft-dropdown, which lacks this feature (or is broken).
Also, tab-ing to the dropdown and trying to use the up-down arrows also doesn't seem to work.

Why: 
When trying to fill out a couple of fields, I often use only the keyboard to `TAB` through the fields, and since the dropdown doesn't work, I need to use the mouse.

**Comment by benhaynes** [comment]
We'll be spending some more time on proper accessibility and keyboard shortcuts soon... it's bittersweet building your own inputs, way more control, but you have to rebuild all the important system stuff too. Any PRs are welcome in the meantime! 😄

**Reply by Fusseldieb** [reply]
> it's bittersweet building your own inputs, way more control, but you have to rebuild all the important system stuff too.

Yea, I felt that. I once tried to make a Select control from scratch (in Vue). It can be a nightmare.
However, Directus v9 is absolutely _gorgeous_ in its design, not gonna lie! In the end it's totally worth it :)

> Any PRs are welcome in the meantime! 😄

Nice to know, I hope that in the next couple of days I have some time to do so!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
